uint32 HBMSOC               #动力电池电量
uint8 Pbrk_act             #驻车制动状态
bool Vehicle_model                 #车辆模式 0--有人 1--无人
uint8 Cyclic_check         #0-15循环校验
uint32 HBM_voltage          #动力电池电压
uint16 Water_temp           #发动机水温
float32 Vehicle_speed_Kmh            #车速
uint8 Gear_position        #挡位
uint8 Sysflt               #车辆故障码 0x01-P;2-R;3-N;4-D;5-WP;6-WD
uint8 Vehicle_state        #车辆状态码
uint32 Whl_spd_l_rpm   #左电机转速
uint32 Whl_spd_r_rpm   #右电机转速
uint32 Eng_spd_rpm     #发动机转速
uint8 EHB_actual_pedal     #当前制动踏板开度
uint32 Pump_spd_rpm         #水泵电机转速
